apollo-adminservice/src/main/scripts/startup.sh

```diff
@@ -63,6 +67,22 @@ function checkPidAlive {
     exit 1;
 }
 
+function existProcessUsePort() {
+    if [ "$(lsof -i:$SERVER_PORT -sTCP:LISTEN -t)" != "" ]; then
```

Comment:
I'm not sure whether the default apollo docker image has the `lsof` command, so it's better to do a test.

Reply:
apollo docker image exists `lsof`

```bash
bash-4.3# ls /usr/bin/ | grep lsof
lsof
```

but it seems didn't work as expect.
```bash
bash-4.3# lsof -i:8080 -sTCP:LISTEN -t
1       /bin/bash       /dev/null
1       /bin/bash       pipe:[288541]
1       /bin/bash       pipe:[288542]
1       /bin/bash       /apollo-quick-start/demo.sh
106     /bin/bash       /dev/pts/0
106     /bin/bash       /dev/pts/0
106     /bin/bash       /dev/pts/0
106     /bin/bash       /dev/pts/0
218     /bin/busybox    /dev/null
218     /bin/busybox    pipe:[288541]
218     /bin/busybox    pipe:[288542]
```

Reply:
This looks like the quick start docker image, would you please check the [standalone one](https://hub.docker.com/r/apolloconfig/apollo-configservice)?

Reply:
There is no `lsof` in [standalone one](https://hub.docker.com/r/apolloconfig/apollo-configservice)

---

check by
```bash
docker pull apolloconfig/apollo-configservice
docker run -it apolloconfig/apollo-configservice:latest bash
```

then run 
```bash
ls /usr/bin/ | grep lsof
lsof -i:8080 -sTCP:LISTEN -t
```
the result is
```bash
ubuntu18@LAPTOP-PJ6N2E8S:~$ docker run -it apolloconfig/apollo-configservice:latest bash
root@a3ad1e15c509:/# ls /usr/bin/ | grep lsof
root@a3ad1e15c509:/# lsof -i:8080 -sTCP:LISTEN -t
bash: lsof: command not found
```

---

It seems that `curl` would be more general than `lsof`?

Reply:
Looks like so, or we may install the lsof command in the image?

Reply:
We can install `lsof` in the image, but this `startup.sh` maybe run on user's machine...

If `lsof` doesn't exists in user's machine, `startup.sh` will fail.

Command `curl` has been use in the old version `startup.sh` already, so can we keep it in this pr?

We can use `lsof` in the future, and introduce a fallback policy when `lsof` doesn't exists. i.e use `curl` or another command to instread of `lsof`.

Reply:
You are right, I would suggest we keep using `curl` for better backward-compatibility.